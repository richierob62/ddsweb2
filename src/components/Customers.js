import React from 'react';
import CustomerList from './CustomerList';
import Details from './Details';
import ContextMenu from './ContextMenu';

const Customers = () => {
    return (
        <div className="Customers">
            <h1 className="page-title">Customers</h1>
            <div className="Page-vertical">
                <CustomerList />
                <div className="Page-lower">
                    <Details />
                    <ContextMenu />
                </div>
            </div>            
        </div>
    );
}

export default Customers;