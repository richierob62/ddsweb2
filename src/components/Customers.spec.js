import React from 'react';
import Customers from './Customers';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

it('renders without crashing', () => {
  const el = shallow(<Customers />);
  expect(shallowToJson(el)).toMatchSnapshot();
});

