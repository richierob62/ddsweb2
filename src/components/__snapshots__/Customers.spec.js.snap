exports[`test renders without crashing 1`] = `
<div
  className="Customers">
  <h1
    className="page-title">
    Customers
  </h1>
  <div
    className="Page-vertical">
    <Connect(customerTable) />
    <div
      className="Page-lower">
      <Details />
      <ContextMenu />
    </div>
  </div>
</div>
`;
